mppm/sys-req.md

```diff
@@ -1,17 +1,11 @@
 ---
 id: sys-req
-title: System requirements and compatibility
+title: Compatibility
 ---
-
-The Multiplayer Play Mode package requires the following:
```

Comment:
Does the removal of this section mean there are no version requirements for Multiplayer Play Mode?

Reply:
There are visible inconsistencies between https://docs-multiplayer.unity3d.com/mppm/current/about/ and https://docs-multiplayer.unity3d.com/mppm/current/sys-req/. For some reason, it seems it was not caught. This PR is to fix this.